Source/UIKit/UIControl.swift

```diff
@@ -20,6 +21,21 @@ extension UIControl {
             .map { $0 as? UIControl }
             .flatMapError { _ in SignalProducer(value: nil) }
     }
+
+    /// Creates a bindable property to wrap a control's value.
+    /// 
+    /// This property uses `UIControlEvents.ValueChanged` and `UIControlEvents.EditingChanged` 
+    /// events to detect changes and keep the value up-to-date.
+    //
+    @warn_unused_result(message="Did you forget to use the property?")
+    class func rex_valueChanged<Host: UIControl, T>(host: Host, getter: Host -> T, setter: (Host, T) -> ()) -> MutableProperty<T> {
```

Comment:
I agree that the name is not entirely correct mainly because this is a property with a state so the _Changed_ part does not make much sense. I would suggest `rex_value` which I think is more accurate since is a bindable property for a value of a `UIControl`.

The name will clash with `UISlider`'s `value` (in terms of naming only) but this property is not intended to be public, it's only a convenient method to be (re-)used internally. But I'm totally open for other suggestions.